st2api/st2api/controllers/v1/webhooks.py

```diff
@@ -110,6 +115,21 @@ def post(self, *args, **kwargs):
 
         return body
 
+    def _parse_request_body(self, content_type, body):
+        if content_type == 'application/json':
+            self._log_request('Parsing body as JSON', request=pecan.request)
+            body = json.loads(body)
+        elif content_type in ['application/x-www-form-urlencoded', 'multipart/form-data']:
+            self._log_request('Parsing body as form encoded data', request=pecan.request)
+            body = urlparse.parse_qs(body)
+        else:
+            # For backward compatibility reasons, try to parse any other content
```

Comment:
I went ahead and made this change.

If it turns out that for some reasons many of our users indeed do not specify a Content-Type header and this breaks it for them, we can consider reverting this change.